gui/default/assets/lang/lang-en.json

```diff
@@ -29,7 +29,6 @@
    "Anonymous Usage Reporting": "Anonymous Usage Reporting",
    "Anonymous usage report format has changed. Would you like to move to the new format?": "Anonymous usage report format has changed. Would you like to move to the new format?",
    "Apply": "Apply",
-   "Are you sure you want to continue?": "Are you sure you want to continue?",
```

Comment:
This one for example looks like it might be a good candidate for reusing. I'd rather waste some translators' time on these few strings than dumping the effort of past contributors.

If it were easy to resurrect the translations from Git history later, this would not matter much. But I'm afraid it won't be that easy, especially with Transifex involved.

Reply:
Yeah, there are some that perhaps may re-appear later, but when it comes to the longer sentences, it's rather unlikely. Some of the others are simply outdated, e.g. the 2014-2019 copyright notice.

For the record, the translate script doesn't remove unused strings automatically, but there has already been a case historically where all unused strings were removed, and that happened in https://github.com/syncthing/syncthing/commit/ec718e729e5cc0dbba5c7f2535051b2bf6e5df3a. The strings listed to remove in this PR are those that have become obsolete since that time point.